lib/ruby_lsp/ruby_lsp_rails/server.rb

```diff
@@ -4,6 +4,15 @@
 require "json"
 require "open3"
 
+# This replicates what Rails does internally
+if defined?(ActiveRecord::Base)
+  ActiveSupport.on_load(:active_support_test_case) do
+    include ActiveRecord::TestFixtures
+
+    fixture_paths << "#{Rails.root}/test/fixtures/"
```

Comment:
Where do people normally configure their `fixture_paths`?

I just realized that, if people do it in `test/test_helper.rb` or in `environments/test.rb`, we are probably going to miss it anyway.

I may have set you on the wrong path here, I'm not sure we can accurately detect which fixture paths were configured.

Also, the `server.rb` file is loaded after the Rails app is fully initialized, so any initializers would have already run.